LightVox_UnitTests/stdafx.cpp

```diff
@@ -0,0 +1,8 @@
+// stdafx.cpp : source file that includes just the standard includes
+// LightVox_UnitTests.pch will be the pre-compiled header
+// stdafx.obj will contain the pre-compiled type information
+
+#include "stdafx.h"
```

Comment:
Could you explain why we need the .cpp file here?

Reply:
we dont